q2cli/tools.py

```diff
@@ -37,6 +37,21 @@ def export_data(path, output_dir):
     result.export_data(output_dir)
 
 
+def show_importable_types(ctx, param, value):
+    import qiime2.sdk
+
+    PluginManager = qiime2.sdk.PluginManager()
```

Comment:
This is a minor stylistic comment: pep8 recommends using snake case for variables (i.e. lowercase names with words separated by underscores), so a more appropriate name would be something like `plugin_manager`. It's also a little clearer because we're not reusing the class name (`PluginManager`) to represent its instantiation.

That being said, the code can actually be shortened because you don't need a `PluginManager` object to hang around (it's only used once to get the importable types). Here's one way to shorten the code. Also note that I'm using Python's custom [sorting API](https://wiki.python.org/moin/HowTo/Sorting) to avoid taking multiple passes through the `set` (this is faster and more Pythonic).

```python
importable_types = sorted(qiime2.sdk.PluginManager().importable_types, key=repr)
```

You can then loop over this sorted list and print the types with `click.echo`.